packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -165,9 +176,16 @@ class PouchManager {
         getReplicationURL
       ).then(res => {
         this.addSyncedDoctype(doctype)
-
         logger.log('PouchManager: Replication for ' + doctype + ' ended', res)
-
+        //We want to warmupQueries only once by "start". aka we want to warmup
+        //every time we launch the client (app) but not every time a replication
+        //is done (aka every 30s at least)
+        if (
+          !this.warmedUpQueries[doctype] &&
+          replicationOptions.warmupQueries
+        ) {
+          this.warmupQueries(doctype, replicationOptions.warmupQueries)
+        }
```

Comment:
Maybe it would be clearer if this was extracted into `this.checkToWarmupDoctype`, this way, we could put the comment above as the docstring of the function, I think it would be clearer.